woocommerce/wc-api-dev
# Fix for 3.5 Fatal

Currently when running this plugin ( i.e. on an Atomic site, or locally ) alongside Woo 3.5 RC2, the REST API is fataling due to `WC_REST_Dev_Webhook_Deliveries_Controller` no longer being present.

__Recreate Error__
- Install Woo 3.5 RC2
- Install this plugin, and activate it
- Visit `/wp-json` on your site

__Test Fix__
- Install this branch alongside 3.5 RC2
- Visit `/wp-json` and verify all endpoints are listed as expected
